futures-util/src/stream/mod.rs

```diff
@@ -128,7 +128,7 @@ cfg_target_has_atomic! {
     #[cfg(feature = "alloc")]
     mod futures_unordered;
     #[cfg(feature = "alloc")]
-    pub use self::futures_unordered::{futures_unordered, FuturesUnordered};
+    pub use self::futures_unordered::{futures_unordered, FuturesUnordered, IterMut, IterPinMut};
```

Comment:
If we're going to expose these at top-level, IMO they need less generic names (`FuturesUnorderedIterMut` or something like that)

Reply:
Indeed!